docs/deployment/upgrade.md

```diff
@@ -8,53 +8,60 @@ hide_title: true
+When breaking changes have been introduced, the Cumulus Core team will publish instructions on migrating from one version to another.  Detailed release notes with migration instructions (if any) for each release can be found on the [Cumulus Releases] page.
 
-Detailed release notes with migration instructions (if any) for each release can be found on the [Cumulus releases page](https://github.com/nasa/cumulus/releases).
+**IMPORTANT** 
 
-**IMPORTANT:** When upgrading through many versions, each migration should be done in the order of release (if going from version 1.1.0 from 1.3.0, upgrade from 1.1.0 to 1.2.0 and then to 1.3.0).
+1. **Use consistent Cumulus versions:** All modules must be updated to the same Cumulus version number (see below)
+2. **Follow all intervening steps:** When skipping over versions, you **must perform all intervening migration steps**.  For example, if going from version 1.1.0 to 1.3.0, upgrade from 1.1.0 to 1.2.0 and then to 1.3.0.  This is critical because each release that contains migration steps provide instructions _only_ for migrating from the _immediately_ previous release, but you must follow _all_ migration steps between your currently installed release and _every release_ through the release that you wish to migrate to.
+3. **Migrate lower environments first:** Migrate your "lowest" environment first and test it to ensure correctness before performing migration steps in each successively higher environment.  For example, in the EDCloud progress from a sandbox environment to UAT to SIT to PROD.
+4. **Migrate during appropriate times:** Choose a time to migrate when support is more likely to be available in case you encounter problems, such as when you are most likely to quickly find help in the #cumulus-internal Slack channel.  Prefer earlier in the week over later in the week (particularly avoiding Fridays, if possible).
 
-## Updating Cumulus version
+## Updating Cumulus Version
 
 To update your Cumulus version:
 
-1. Find the desired release on the [Cumulus releases page](https://github.com/nasa/cumulus/releases)
-2. Update the `source` in your Terraform deployment files **for each of [your Cumulus modules](./components.md#available-cumulus-components)** by replacing `vx.x.x` with the desired version of Cumulus:
+1. Find the desired release on the [Cumulus Releases] page
+2. Update the `source` in your Terraform deployment files **for each of [your Cumulus modules](./components.md#available-cumulus-components)** by replacing `vx.x.x` with the desired version of Cumulus.  For example, here's the
+entry from the `data-persistence` module:
 
     `source = "https://github.com/nasa/cumulus/releases/download/vx.x.x/terraform.zip//tf-modules/data-persistence"`
 
 3. Run `terraform init` to get the latest copies of your updated modules
 
-## Update data persistence resources
+## Update Data Persistence Resources
```

Comment:
Capitalizing these headers is inconsistent with how we've been updating our docs recently (we've updated PRs the other way ... often)   We should have a conversation about the right standard.     No strong feelings from me other than ensuring consistency moving forward.

Reply:
No worries. I like to be consistent, so I'll revert this change. (As an aside, I do prefer title-cased titles, but consistency trumps preferences.)